ci/jenkins/jobs/projects-lab.yaml

```diff
@@ -906,6 +906,38 @@
             - text:
                 credential-id: DOCKER_PASSWORD
                 variable: DOCKER_PASSWORD
+      - '{name}-{test_name}-for-pull-request':
```

Comment:
As we discussed, the Jenkins kind job will eventually be migrated to AWS as cloud jobs. Would you like to implement it in this PR or in another one? I am okay with either.  cc @rajnkamr

Reply:
@XinShuYang 
We should go ahead without jenkins kind cloud job support in this PR.
Will raise another PR for same.

Reply:
> @XinShuYang We should go ahead without jenkins kind cloud job support in this PR. Will raise another PR for same.

Got it, we can create an issue to track this migration.

Reply:
#6513 is created to track cloud jenkins support in kind.